recipes/uwsgi/meta.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+  name: uwsgi
```

Comment:
Both conda and PyPI are case insensitive when installing and searching for packages. So I guess it does not really matter. I would use `uWSGI`, but I am OK with `uwsgi`.